package ExceptionHandling.Exceptions.UserDefined;
public class NewException extends Exception {
    NewException() {
        super("This is an NewException which is created by User and it's also called as UserDefined Exception");
    }
}
